presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -406,6 +412,26 @@ private Object toPrestoValue(Object tpchValue, Type columnType)
         throw new IllegalArgumentException("unsupported column type " + columnType);
     }
 
+    @Override
+    public ConnectorAnalyzeTableMetadata getAnalyzeTableMetadata(ConnectorSession session, ConnectorTableHandle tableHandle, Map<String, Object> properties)
+    {
+        ConnectorTableMetadata tableMetadata = getTableMetadata(session, tableHandle);
+        TableStatisticsMetadata statisticsMetadata = new TableStatisticsMetadata(ImmutableSet.of(), ImmutableSet.of(ROW_COUNT), ImmutableList.of());
+        return new ConnectorAnalyzeTableMetadata(statisticsMetadata, tableMetadata, tableHandle);
+    }
+
+    @Override
+    public ConnectorAnalyzeTableHandle beginAnalyze(ConnectorSession session, ConnectorTableHandle tableHandle)
+    {
+        return (TpchTableHandle) tableHandle;
+    }
+
+    @Override
+    public void finishAnalyze(ConnectorSession session, ConnectorAnalyzeTableHandle tableHandle, Collection<ComputedStatistics> computedStatistics)
+    {
+        // do nothing
```

Comment:
verify `computedStatistics` contains ROW COUNT only and the ROW COUNT matches. -- as you consider this code path to be "for tests only", adding some verification looks natural, right?